README.md

```diff
@@ -76,42 +73,15 @@ This means you can know who is using your app, how they're using it, and where y
 
 PostHog is the only **product-focused** open-source analytics library, with an event and user-driven architecture. That means tracking identifiable (where applicable) user behavior, and creating user profiles. We are an open-source alternative to products such as Mixpanel, Amplitude, or Heap, that's designed to be more developer-friendly than them, with a broader range of features like session recording, heatmaps, feature flags and plugins.
 
-There are a few session-based open-source libraries that are nice alternatives to Google Analytics. That's not what we are focused on. 
+We play nicely with data warehouses - you can _import_ event or user data by writing a plugin to create transformations, or you can _export_ data by using our existing data exports to BigQuery, Redshift, Snowflake, etc. Uniquely, you can host PostHog in the same infrastructure.
 
 <br /><br />
 
 <img align="left" src="https://user-images.githubusercontent.com/65415371/124740386-ab395a00-df12-11eb-8519-3cd1b26f9509.png" width="50px" />
 
 ## PostHog Cloud
 
-You can [sign up for a free account](https://app.posthog.com/signup) on our hosted platform.
-
-<br /><br />
-
-<img align="left" src="https://user-images.githubusercontent.com/65415371/124740447-b7bdb280-df12-11eb-80c3-b2aa85be0f86.png" width="50px" />
-
-## Deployment Options
-
-Our suggested method for quick deployment is Heroku's one-click option:
-
-[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/posthog/posthog)
-
-However, PostHog can be deployed anywhere you want! Here are step-by-step tutorials we've written for deployment options using all the major cloud providers:
-
-* [AWS](https://posthog.com/docs/deployment/deploy-aws)
-* [Microsoft Azure](https://posthog.com/docs/deployment/deploy-azure)
-* [Digital Ocean](https://posthog.com/docs/deployment/deploy-digital-ocean)
-* [Google Cloud](https://posthog.com/docs/deployment/deploy-gcs)
-* [Linode](https://posthog.com/docs/deployment/deploy-linode)
-* [Render](https://posthog.com/docs/deployment/deploy-render)
-
-<br /><br />
-
-<img align="left" src="https://user-images.githubusercontent.com/65415371/124740742-f94e5d80-df12-11eb-8ca9-7d2ad4f56e3d.png" width="50px" />
-
-## Production Deployment
-
-[See our Docs for production deployment options.](https://posthog.com/docs/deployment)
+You can [sign up for a free account](https://app.posthog.com/signup) if you don't want to host PostHog yourself.
```

Comment:
Shouldn't we still have a self hosting section here too?
```suggestion
You can [sign up for a free account](https://app.posthog.com/signup) if you don't want to host PostHog yourself.

<br /><br />

## Self-hosting

[See our Docs for self-hosted production deployment options.](https://posthog.com/docs/deployment)
```

Reply:
agreed - was weird to callout cloud so specifically down here, changed this section to re-iterate the points from "Quick start"